### ISSUE TITLE
Fixes to run the image on Windows with Linux images

### ISSUE THREAD

**Issue by plindman** [smart-on-fhir/smart-local-sandbox#1]
To get the image working on Windows with Linux Images (default Docker for Win settings), you need to do the following: 

1. Ensure line endings are not \n (and not Windows \r\n).  

When cloning the repo, add switch `--config core.autocrlf=input.`  
`> git clone https://github.com/smart-on-fhir/smart-local-sandbox.git  --config core.autocrlf=input`

The source for this tip and more details can be found here:  
[https://willi.am/blog/2016/08/11/docker-for-windows-dealing-with-windows-line-endings/](https://willi.am/blog/2016/08/11/docker-for-windows-dealing-with-windows-line-endings/)

2. Remove the path mappings in run.bat  

```
-v %CWD%/backend/:/usr/app/backend ^
-v %CWD%/ext/hapi-dstu2:/usr/app/ext/hapi-dstu2 ^
-v %CWD%/ext/hapi-dstu3:/usr/app/ext/hapi-dstu3 ^
```

**run_winlin.bat:**
```
@ECHO OFF
SET CWD=%cd%  

docker run -it ^
    -p 4000:4000 ^
    -p 4001:4001 ^
    -p 4002:4002 ^
    -p 4003:4003 ^
    -p 4004:4004 ^
    smart-sandbox:latest bash
```

**Comment by vlad-ignatov** [comment]
That is correct. In fact we have abandoned this project for now. Please check out https://github.com/smart-on-fhir/smart-dev-sandbox instead.
